helm/examples/azure/sa/create-sa.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+
+if [[ -z "$SA_RESOURCE_GROUP" ]] || [[ -z "$SA_LOCATION" ]] || [[ -z "$STORAGE_ACCOUNT_NAME" ]]
+then
+   echo "Miss one of SA_RESOURCE_GROUP, SA_LOCATION and STORAGE_ACCOUNT_NAME"
+   exit 1
+fi
+
+[[ -n "$SUBSCRIPTION" ]] && az account set --subscription $SUBSCRIPTION
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SECRET_NAME_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SECRET_NAMESPACE_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*secretNamespace:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SHARE_NAME_INPUT=$(cat $VALUES_FILE | grep "^[[:space:]]*shareName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+[[ -n "$SECRET_NAME_INPUT" ]] && SECRET_NAME=${SECRET_NAME_INPUT}
+[[ -n "$SECRET_NAMESPACE_INPUT" ]] && SECRET_NAMESPACE=${SECRET_NAMESPACE_INPUT}
+[[ -n "$SHARE_NAME_INPUT" ]] && SHARE_NAME=${SHARE_NAME_INPUT}
+
+if [[ -z "$SECRET_NAME" ]] || [[ -z "$SECRET_NAMESPACE" ]] || [[ -z "$SHARE_NAME" ]]
+then
+   echo "Miss one of SECRET_NAME, SECRET_NAMESPACE and SHARE_NAME"
+   exit 1
+fi
+
+# Create a resource group
+# Check Resource Group
+rc=$(az group exists --name ${SA_RESOURCE_GROUP})
+if [ "$rc" != "true" ]
+then
+  az group create --name ${SA_RESOURCE_GROUP} --location ${SA_LOCATION}
+fi
+
+az storage account check-name -n $STORAGE_ACCOUNT_NAME | \
+	grep -q "\"nameAvailable\":[[:space:]]*true"
+if [ $? -ne 0 ]
```

Comment:
The grep above this line should probably be for "not a valid storage account name" and this check changed to -eq.  A pre-existing storage account will fail this, as-is, because the name is not available.

Reply:
I remove this section since nameAvailabe return "false" if exists.